client/driver/java.go

```diff
@@ -133,8 +133,17 @@ func (d *JavaDriver) Start(ctx *ExecContext, task *structs.Task) (DriverHandle,
 	// Get the environment variables.
 	envVars := TaskEnvironmentVariables(ctx, task)
 
+        // Look for jvm options
```

Comment:
Hey, I think we can simplify the code if we put

`var jvm_options []string` above the if and then we can change the if statement to be the positive case and we can append there.